Objects/dictobject.c

```diff
@@ -1506,6 +1597,20 @@ _Py_dict_lookup_threadsafe(PyDictObject *mp, PyObject *key, Py_hash_t hash, PyOb
     return ix;
 }
 
+Py_ssize_t
+_Py_dict_lookup_threadsafe_stackref(PyDictObject *mp, PyObject *key, Py_hash_t hash, _PyStackRef *value_addr)
+{
+    PyObject *val;
+    Py_ssize_t ix = _Py_dict_lookup(mp, key, hash, &val);
+    if (val != NULL) {
+        *value_addr = PyStackRef_FromPyObjectNew(val);
+    }
+    else {
+        *value_addr = PyStackRef_NULL;
+    }
```

Comment:
```suggestion
	*value_addr = value == NULL ? PyStackRef_NULL : PyStackRef_FromPyObjectNew(value);
```

(You were using ternary for the inlined function)

Reply:
Ah I'm sorry that I had a typo in my suggestion..,. should have been `val` instead of `value` :(